fetch.bs

```diff
@@ -11,15 +11,28 @@ Translate IDs: typedefdef-bodyinit bodyinit,dictdef-requestinit requestinit,type
 </pre>
 
 <pre class=anchors>
-url:https://tools.ietf.org/html/rfc7230#section-3.1.1;text:method;type:dfn;spec:http
-url:https://tools.ietf.org/html/rfc7230#section-3.2;text:field-name;type:dfn;spec:http
-url:https://tools.ietf.org/html/rfc7230#section-3.2;text:field-content;type:dfn;spec:http
-url:https://tools.ietf.org/html/rfc7230#section-3.2;text:field-value;type:dfn;spec:http
-url:https://tools.ietf.org/html/rfc7230#section-3.1.2;text:reason-phrase;type:dfn;spec:http
+urlPrefix:https://tools.ietf.org/html/rfc7230#;type:dfn;spec:http
+    url:section-3.1.1;text:method
+    url:section-3.2;text:field-name
+    url:section-3.2;text:field-content
+    url:section-3.2;text:field-value
+    url:section-3.1.2;text:reason-phrase
+
 url:https://tools.ietf.org/html/rfc7234#section-1.2.1;text:delta-seconds;type:dfn;spec:http-caching
-url:https://tools.ietf.org/html/rfc8941#section-2;text:structured field value;type:dfn;spec:rfc8941
-url:https://tools.ietf.org/html/rfc8941#section-4.1;text:serializing structured fields;type:dfn;spec:rfc8941
-url:https://tools.ietf.org/html/rfc8941#section-4.2;text:parsing structured fields;type:dfn;spec:rfc8941
+
+urlPrefix:https://tools.ietf.org/html/rfc8941#;type:dfn;spec:rfc8941
+    url:section-2;text:structured field value
+    url:section-4.1;text:serializing structured fields
+    url:section-4.2;text:parsing structured fields
+
+url:https://w3c.github.io/resource-timing/#dfn-mark-resource-timing;text:mark resource timing;type:dfn;spec:resource-timing
+
+urlPrefix:https://w3c.github.io/hr-time/#;spec:hr-time
+    type:dfn
+        url:dfn-coarsen-time;text:coarsen time
+        url:dfn-coarsened-shared-current-time;text:coarsened shared current time
+        url:dfn-unsafe-shared-current-time;text:unsafe shared current time
+    type:idl;url:dom-domhighrestimestamp;text:DOMHighResTimeStamp
```

Comment:
"idl" is a linking supertype, not a valid type for the dfns themselves. Bikeshed not complaining about that is my fault; filed <https://github.com/tabatkins/bikeshed/issues/2068>

```suggestion
    type:typedef;url:dom-domhighrestimestamp;text:DOMHighResTimeStamp
```